pkgs/pre-commit-check/default.nix

```diff
@@ -89,7 +89,9 @@ pre-commit-hooks.lib.${system}.run {
               if [ "$PRE_COMMIT" = "1" ] \
               && [ -n "$PRE_COMMIT_FROM_REF" ] \
               && [ -n "$PRE_COMMIT_TO_REF" ]; then
-                ${config.hooks.commitizen.package}/bin/cz check --rev-range "$PRE_COMMIT_FROM_REF".."$PRE_COMMIT_TO_REF"
+                ${config.hooks.commitizen.package}/bin/cz check \
+                  --allowed-prefixes "Revert" \
+                  --rev-range "$PRE_COMMIT_FROM_REF".."$PRE_COMMIT_TO_REF"
```

Comment:
This will make a lot of PRs red, right? Not the worst thing but also kind of annoying. Would it make more sense to only run this in the merge queue?

Reply:
I would find this pretty annoying. We have a handful of draft and "always open" PRs and one of the ways I narrow down what to pick up for review is the green check mark.